### PR TITLE
[DOCS] Expands transforms docs with CCS info

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR adds information on CCS support to transforms docs:
* a note on CCS is supported to the `source` setting of the PUT/Preview/cat/Update Transforms API
* a note on how to set the remote clusters if you use remote indices,
* additional CCS info related to runtime mappings
* a limitation item on CCS.

### Preview

[PUT Transforms API](https://elasticsearch_70156.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/put-transform.html)
[Limitations](https://elasticsearch_70156.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/transform-limitations.html#transforms-ccs-limitation)

### CODE REVIEW

File: docs/reference/rest-api/common-parms.asciidoc [hunk 1]
```diff
@@ -918,13 +918,18 @@ end::source_includes[]
 
 tag::source-transforms[]
 The source of the data for the {transform}.
+
+INFO: {transforms-cap} support <<modules-cross-cluster-search,{ccs}>>.
```
Comment (szabosteve) [inline docs/reference/rest-api/common-parms.asciidoc:922]:
```suggestion
NOTE: {transforms-cap} support <<modules-cross-cluster-search,{ccs}>>.
```

File: docs/reference/rest-api/common-parms.asciidoc [hunk 2]
```diff
@@ -933,7 +940,9 @@ A query clause that retrieves a subset of data from the source index. See
 end::source-query-transforms[]
 
 tag::source-runtime-mappings-transforms[]
-Definitions of search-time runtime fields that can be used by the transform.
+Definitions of search-time runtime fields that can be used by the transform. If 
+you use <<modules-cross-cluster-search,{ccs}>>, the version of the remote 
+cluster must be newer than 7.12.
```
Comment (lcawl) [inline docs/reference/rest-api/common-parms.asciidoc:945]: Is it 7.12 and higher? Or just 7.13?  If the former, I suggest:
```suggestion
cluster must be 7.12 or later.
```

File: docs/reference/transform/limitations.asciidoc [hunk 3]
```diff
@@ -275,3 +275,13 @@ accessed. If the time field is a {ref}/runtime.html[runtime field], the
...
 {transforms}.
+
+
+[discrete]
+[[transforms-ccs-limitation]]
+=== {transforms-cap} supports {ccs} if the remote cluster configured properly
+
+If you use <<modules-cross-cluster-search,{ccs}>>, the remote cluster must 
+support the search and aggregations you use in your {transforms}. 
+{transforms-cap} validate their configuration, if it fails and you use {ccs},
```
Comment (lcawl) [inline docs/reference/transform/limitations.asciidoc:286]: Not mandatory, but I wasn't sure if "it" referred to the transform or the validation or the configuration:
```suggestion
{transforms-cap} validate their configuration; if you use {ccs} and the validation fails, 
```

File: docs/reference/transform/limitations.asciidoc [hunk 4]
```diff
@@ -275,3 +275,13 @@ accessed. If the time field is a {ref}/runtime.html[runtime field], the
 performance impact of calculating field values at query time can significantly 
 slow the {transform}. Use an indexed field as a time field when using 
 {transforms}.
+
+
+[discrete]
+[[transforms-ccs-limitation]]
+=== {transforms-cap} supports {ccs} if the remote cluster configured properly
```
Comment (lcawl) [inline docs/reference/transform/limitations.asciidoc:282]:
```suggestion
=== {transforms-cap} support {ccs} if the remote cluster is configured properly
```

**Review by lcawl — APPROVED** [review]
Minor comments, otherwise LGTM

File: docs/reference/rest-api/common-parms.asciidoc (hunk 2 above)
Comment (szabosteve) [reply docs/reference/rest-api/common-parms.asciidoc:945]: The issue says "the remote must be > 7.12", so I assume it has to be at least 7.13. @hendrikmuhs can you please confirm it?

Comment (hendrikmuhs) [reply docs/reference/rest-api/common-parms.asciidoc:945]: its 7.12 or later.

I think it sounds nicer if you say: For search runtime fields all data nodes, including remote nodes, must be 7.12 or later.

**Review by hendrikmuhs — COMMENTED** [review]
I added some comments

**Comment by szabosteve** [comment]
@hendrikmuhs I addressed your comments and suggestions via 9f63fa1.

**Review by hendrikmuhs — APPROVED** [review]
LGTM

**Pull request merged**